tests/zenko_tests/node_tests/backbeat/tests/ingestion/ingestionS3CPauseResume.js

```diff
@@ -0,0 +1,116 @@
+const assert = require('assert');
+const  { series } = require('async');
+const uuid = require('uuid/v4');
+
+const { scalityS3Client, ringS3Client } = require('../../../s3SDK');
+const IngestionUtility = require('../../IngestionUtility');
+const BackbeatAPIUtility = require('../../BackbeatAPIUtility');
+
+const scalityUtils = new IngestionUtility(scalityS3Client, ringS3Client);
+const ringS3CUtils = new IngestionUtility(ringS3Client);
+const backbeatAPIUtils = new BackbeatAPIUtility();
+const ingestionSrcBucket = process.env.RING_S3C_INGESTION_SRC_BUCKET_NAME;
+const srcLocation = process.env.RING_S3C_BACKEND_SOURCE_LOCATION;
+const location = srcLocation;
+let INGESTION_DEST_BUCKET;
+let KEY_PREFIX;
+let OBJ_KEY1;
+let OBJ_KEY2;
+let OBJ_DATA;
+const INGESTION_TIMEOUT = 300000;
+
+describe('Ingestion pause resume', function() {
+    this.timeout(INGESTION_TIMEOUT);
+
+    beforeEach(done => {
+        INGESTION_DEST_BUCKET = `ingestion-dest-bucket-${uuid()}`;
+        KEY_PREFIX = `${ingestionSrcBucket}-${uuid()}`;
+        OBJ_KEY1 = `${KEY_PREFIX}/object-to-ingest-${uuid()}`;
+        OBJ_KEY2 = `${KEY_PREFIX}/object-to-ingest-${uuid()}`;
+        scalityUtils.createIngestionBucket(INGESTION_DEST_BUCKET, location, err =>{
+            assert.ifError(err);
+            return done();
+        });
```

Comment:
```suggestion
        scalityUtils.createIngestionBucket(INGESTION_DEST_BUCKET, location, done);
```
passing in done will do the same thing